worker/src/RTC/Consumer.cpp

```diff
@@ -267,6 +272,38 @@ namespace RTC
 			return;
 		}
 
+		// Check whether sequence number and timestamp sync is required.
+		if (this->syncRequired)
+		{
+			this->seqNum += 1;
+
+			uint64_t now = DepLibUV::GetTime();
+
+			if (now > this->rtpTimestamp)
+				this->rtpTimestamp = now;
+
+			this->syncRequired = false;
+
+			RequestFullFrame();
```

Comment:
Is this call required? Note than, upon resumed, the Consumer already requests a full frame.

Reply:
Does it also cover the case where the remote endpoint changes the video device or updates the resolution?

Reply:
If the remote Producer changes the video stream then it will send a full frame by itself.

Reply:
Okey

Reply:
For some reason requesting it from the Producer does not work 100% or times. It does when requesting from Cosumer.

I'll investigate further to know the reason. Meanwhile it's functional.